origin.spec

```diff
@@ -46,16 +46,16 @@
 %endif
 %{!?make_redistributable: %global make_redistributable %{need_redistributable_set}}
 
-%if "%{dist}" == ".el7aos"
+%if 0%{?epel}
 %global package_name openshift
 %global product_name OpenShift
 %else
-%global package_name openshift
-%global product_name OpenShift
+%global package_name okd
+%global product_name OKD
 %endif
 
-%{!?version: %global version 0.0.1}
-%{!?release: %global release 1}
+%{!?version: %global version 4.1.0}
+%{!?release: %global release 1.1417.98b35a9}
```

Comment:
Are you generating this?  It would change on ever commit if so which won’t work.

Reply:
Yeah..I realized after the fact that committing the vars can never work with CI.
The right way is probably to have a proper make target for OKD builds and pass the vars at build time in a proper CI pipeline.

Reply:
Removed the spec update generation.